app/services/idv/cancel_verification_attempt.rb

```diff
@@ -7,11 +7,14 @@ def initialize(user:)
     end
 
     def call
-      user.profiles.gpo_verification_pending.each do |profile|
-        profile.update!(
-          active: false,
-          deactivation_reason: :verification_cancelled,
-        )
+      user.profiles.each do |profile|
+        if profile.gpo_verification_pending?
+          profile.update!(
+            active: false,
+            deactivation_reason: :verification_cancelled,
+            gpo_verification_pending_at: nil,
```

Comment:
Do we want to set anything to do with the fraud timestamps here as well? Or are we keeping whatever is there, even if the user cancels?

Reply:
That's a good question that i'll find the answer to.